### ISSUE TITLE
Switch to another highlighter library

### ISSUE THREAD

**Issue by xmatthewx** [mmmavis/docs-analyzer#14]
Seems like a bug was introduced. After loading, text on the page gains a drop shadow. Some links move around. Experienced this on Chrome, but not Canary. @flukeout reported it on Firefox as well. 

@flukeout, can you add anymore to clarify?

**Comment by flukeout** [comment]
Here's the logo flashing in Firefox...

![flash](https://cloud.githubusercontent.com/assets/25212/16926089/1bb00816-4cdc-11e6-9275-675348f84362.gif)

My wild hypothesis is that this is somehow related to to the etherpad cursor, which flashes at the same rate. Somehow someway, etherpad code (including some styling?) is acting on the entire app, causing strange behaviour and slowdown.

**Comment by mmmavis** [comment]
the owner of one of the libraries I included doesn't like his library being hotlinked. That's why he injects `evil.css` to websites that are hotlinking to his `.js` file. I tried linking to a local copy on the our app's server instead but it didn't solve the problem unless I removed the chunk of code that's making the injection from the library's source code. I'm gonna find another library to use. Blah.
